rebar.config

```diff
@@ -6,7 +6,7 @@
   %%  {folsom, ".*", {git, "https://github.com/cloudant/couchdb-folsom.git", "fbb7bc83806520ffef84107c85f53c1f7113c20d"}},
   %%  Master has dependence on on meck 0.8.2 (only used in folsom testing) which fails under R18
   %%  Branch uses {meck, ".*", {git, "https://github.com/eproxus/meck.git", {tag, "0.8.4"}}}
-  {folsom, ".*", {git, "https://github.com/cloudant/couchdb-folsom.git", {branch, "64114-support-erlang-r18-folsom"}}},
+  {folsom, ".*", {git, "https://github.com/apache/couchdb-folsom.git", {tag, "CouchDB-0.8.3"}}},
```

Comment:
Just an observation for readers: this tag is actually the latest and 9 days old! https://github.com/apache/couchdb-folsom/releases